packages/server/src/worker/workerTypes.ts

```diff
@@ -57,11 +58,26 @@ export interface TJobCheckLinksResult {
   kind: JobKind.CheckLinks;
 }
 
+/**
+ *  Update algolia index with latest resources
+ */
+export interface TJobSyncAlgoliaData {
+  kind: JobKind.SyncAlgolia;
+  userId: string; // _id of user who started this job
+}
+export interface TJobSyncAlgoliaResult {
```

Comment:
Is there an error from the Algolia API that we would want here?

For instance, if we try the sync to algoia and their server is down would we want to have our job succeed (presumably with `wasUpdated` `false`) and report the error back?